ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -447,6 +448,12 @@ export const ListSearch = fp.flow(withCdrVersions(), withCurrentWorkspace(), wit
       return this.props.searchContext.domain === Domain.SURVEY;
     }
 
+    getConceptLink(conceptId) {
+      const link = environment.publicUiUrl + '/ehr/' +
+        domainToTitle(this.props.searchContext.domain).toLowerCase() + '?search=' + conceptId;
+      return link;
```

Comment:
Nit: can you drop the `link` var and just return the string directly?

Reply:
Done